pages/[username]/[slug]/index.public.js

```diff
@@ -29,14 +29,10 @@ export default function Post({ contentFound, rootContentFound, parentContentFoun
   return (
     <>
       {showConfetti && <Confetti />}
-      <DefaultLayout metadata={contentMetadata}>
+      <DefaultLayout as="article" metadata={contentMetadata}>
```

Comment:
Não sei se isso é intencional, mas essa alteração remove o `article` da publicação principal, certo? Ficaria assim:

![Print com o menu de desenvolvedor aberto](https://github.com/filipedeschamps/tabnews.com.br/assets/26308880/574dada6-55b2-4ea4-b587-01a2e663820c)

Se for intencional, você sabe me dizer se o `main` cumpre um papel de "`article` principal"? Relendo a [especificação](https://html.spec.whatwg.org/multipage/sections.html#the-article-element), me parece que _sim_:

> When the main content of the page (i.e. excluding footers, headers, navigation blocks, and sidebars) is all one single self-contained composition, that content may be marked with an [article](https://html.spec.whatwg.org/multipage/sections.html#the-article-element), but it is technically redundant in that case (since it's self-evident that the page is a single composition, as it is a single document).

E, levando isso em consideração, me parece que não precisariamos do `article` nem mesmo na linha que você mencionou, ficando assim, tudo dentro do `main` e os comentários dentro de um único `article`:

![Print](https://github.com/filipedeschamps/tabnews.com.br/assets/26308880/55f1be00-5834-4349-a7d7-ebd6683f9374)

Porém, a remoção do `article` que engloba a publicação principal traz uma outra consequência. Vendo a [especificação](https://html.spec.whatwg.org/multipage/sections.html#the-article-element) do `address`:

> The [address](https://html.spec.whatwg.org/multipage/sections.html#the-address-element) element [represents](https://html.spec.whatwg.org/multipage/dom.html#represents) the contact information for its nearest [article](https://html.spec.whatwg.org/multipage/sections.html#the-article-element) or [body](https://html.spec.whatwg.org/multipage/sections.html#the-body-element) element ancestor. If that is [the body element](https://html.spec.whatwg.org/multipage/dom.html#the-body-element-2), then the contact information applies to the document as a whole.

Esse texto já me deixou com um pé atrás de tirarmos o `article` para deixar apenas o `main`.